evap/contributor/templates/contributor_index.html

```diff
@@ -157,13 +157,36 @@
                                                         <span class="fas fa-pencil"></span>
                                                     </a>
                                                     {% if not evaluation|has_nonresponsible_editor %}
-                                                        <button class="btn btn-sm btn-dark" data-bs-toggle="tooltip"
-                                                            data-bs-placement="top" title="{% translate 'Delegate preparation' %}"
-                                                            data-evaluation-name="{{ evaluation.full_name }}"
-                                                            data-delegation-url="{% url 'contributor:evaluation_direct_delegation' evaluation.id %}"
-                                                        >
-                                                            <span class="fas fa-hand-point-left"></span>
-                                                        </button>
+                                                        <form class="d-inline" method="POST"
+                                                              action="{% url 'contributor:evaluation_direct_delegation' evaluation.id %}">
+                                                            {% csrf_token %}
+
+                                                            <confirmation-modal type="submit">
+                                                                <span slot="title">{% translate 'Delegate preparation' %}</span>
+                                                                <span slot="action-text">{% translate 'Delegate preparation' %}</span>
+                                                                <span slot="question">
+                                                                    {% blocktranslate trimmed %}
+                                                                        Do you really want to delegate the preparation of the evaluation <strong data-label=""></strong>?
+                                                                    {% endblocktranslate %}
+                                                                </span>
+                                                                <div slot="extra-inputs">
+                                                                    <div class="my-4">
+                                                                    {% comment %}
+                                                                        This bootstrap_form magically puts the delegated people into the html form - if and only if the bootstrap_form is contained in the html form. I do not know why.
+                                                                    {% endcomment %}
```

Comment:
This comment confuses me. Unless my memory is messing with me, the dropdown field is part of the modal dialog. Then, it seems natural to me that we'd need to include some HTML rendering of the form in here, as an extra input.